resilience4j-timelimiter/src/test/java/io/github/resilience4j/timelimiter/TimeLimiterEventPublisherTest.java

```diff
@@ -71,7 +71,7 @@ public void shouldConsumeOnTimeoutEvent() {
         timeLimiter.getEventPublisher()
                 .onTimeout(event -> logger.info(event.getEventType().toString()));
         Supplier<CompletableFuture<String>> futureSupplier = () ->
-                CompletableFuture.supplyAsync(this::fail);
+                CompletableFuture.supplyAsync(this::timeout);
```

Comment:
I added `timeout()` because `fail()` doesn't make TIMEOUT event. It makes ERROR event.